<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionSettingFrame</class>
 <widget class="QFrame" name="ActionSettingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="inputButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Set=&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pos                </string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Delay            </string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x1DelayLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x1SpeedLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1PosLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1DelayLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1SpeedLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zPosLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zDelayLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zSpeedLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="gyButton">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="text">
      <string>GY</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="gzButton">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="text">
      <string>GZ</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="gxButton">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="text">
      <string>GX</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x1PosLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
